build.sbt

```diff
@@ -542,20 +543,23 @@ lazy val api = (project in engine("api")).
     }
   )
 
-lazy val securityApi = (project in engine("security-api")).
+lazy val security = (project in engine("security")).
   settings(commonSettings).
   settings(
-    name := "nussknacker-security-api",
+    name := "nussknacker-security",
     libraryDependencies ++= {
       Seq(
         "org.scalatest" %% "scalatest" % scalaTestV % "test",
+        "org.mindrot" % "jbcrypt" % jbcryptV,
         "com.typesafe.akka" %% "akka-http" % akkaHttpV force(),
+        "com.typesafe.akka" %% "akka-stream" % akkaV force(),
```

Comment:
I wonder if we can have all stuff provided in our ui-server in scope "provided" and if this scope will be respected in library using this module... Thanks to this we will be sure that during assembling it won't go to the jar again...